flutter_local_notifications/example/lib/main.dart

```diff
@@ -1289,6 +1301,40 @@ class _HomePageState extends State<HomePage> {
     return filePath;
   }
 
+  Future<void> _showBigPictureNotificationColoringBackground() async {
+    final String largeIconPath = await _downloadAndSaveFile(
```

Comment:
Does demonstrating the `colorized` functionality need to be tied to showing pictures? If not, could this example code snippet be isolated to bare minimum to demonstrate it?